cusub_cortex/cusub_cortex_bringup/launch/cortex.launch

```diff
@@ -1,9 +1,16 @@
 <launch>
   <group ns="leviathan">
 
+    <arg name="config_filename" default="mission_config.yaml"/>
+
     <include file="$(find mapper)/launch/mapper.launch"/>
+
+    <include file="$(find waypoint_navigator)/launch/waypoint_navigator.launch"/>
+
     <!--  State machine closed for construction-->
-    <include file="$(find state_machine)/launch/state_machine.launch"/>
```

Comment:
Not correct state machine launch file, but I fix it in my latest pull. I like your idea of adding the config_filename as a parameter

Reply:
Oh I see you changed it from smach.lauch -> state_machine.launch ....

Reply:
yea, for the name-spacing, I just kinda name spaced it where it was included or as necessary to make it work.